images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -175,6 +175,11 @@ function Get-PackerVersion {
     return "Packer $(packer --version)"
 }
 
+function Get-PassVersion {
+    $passVersion = pass version | Select-String "^=\s+v").Line.Replace('v','') | Take-OutputPart -Part 2
```

Comment:
I think you need one more "(" before `pass version`
Suggest this way to simplify `(pass version | Select-String "v([\d\.]+)").Matches.Value.Replace('v', '')`

Reply:
Done

Reply:
Maybe another linter for powershell scripts would be nice here :)